Packs/ArcherRSA/Integrations/ArcherV2/ArcherV2.py

```diff
@@ -923,9 +923,11 @@ def get_field_value_list_helper(self, child, values_list, depth, parent="root"):
 
     def get_field_value_list(self, field_id, depth=0):
         cache = get_integration_context()
+        cache_key = f"{field_id}__{depth}"
```

Comment:
Lets add a comment explaining this.

Reply:
Added comments about setting and getting from integration context using both field ID and depth